modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -4862,11 +4893,40 @@ type IntegrationTimeCalculationResult {
   # queried mode
   mode: ObservingMode!
 
-  # ITC results
-  result: IntegrationTime!
+  # ITC results, there maybe one per CCD or a single result
+  results: [IntegrationTime!]!
```

Comment:
This is a good idea, to give the results for each CCD.  Should be a way to tie each result back to a particular CCD or is that overkill?  

```
...
  results: [CcdIntegrationTime!]!
...

type CcdIntegrationTime {
  ccd: String!
  time: IntegrationTime!
}
```

Reply:
Alright, it sound like it isn't worth worrying about.